feature-libs/pickup-in-store/occ/model/occ-stock-endpoints.model.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OccEndpoint } from '@spartacus/core';
+
+declare module '@spartacus/core' {
+  interface OccEndpoints {
+    /**
+     * Get a stock level for a product.
+     *
+     * @member {string} [page]
+     */
+    stock?: string | OccEndpoint;
+
+    /**
+     * Get a stock level for a product at a specific store.
+     *
+     * @member {string} [page]
+     */
+    stockAtStore?: string | OccEndpoint;
+
+    /**
+     * Update the delivery option for a cart entry to pickup or delivery.
+     * @member {string} [page]
+     */
+    updateDeliveryOption?: string | OccEndpoint;
```

Comment:
why `updateDeliveryOption` is in this file?

Reply:
removed, thanks